sample/src/main/res/values/dimens.xml

```diff
@@ -1,6 +1,6 @@
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools">
     <!-- Default screen margins, per the Android Design guidelines. -->
-    <dimen name="norm_grid_size">8dp</dimen>
+    <dimen name="norm_grid_size" tools:ignore="UnusedResources">8dp</dimen>
```

Comment:
Why it is ignored?
Shouldn't be just removed if it is really unused? 
Or maybe it is preserved for backwards compatibility?

Reply:
> Why it is ignored?

It's ignored as it's our base grid size. All the others size are derived from this. I'd rather keep it otherwise `doub_grid_size` would make little sense